ui/imports/shared/views/chat/CompactMessageView.qml

```diff
@@ -736,7 +736,7 @@ Item {
     }
 
     HoverHandler {
-        enabled: !activityCenterMessage &&
+        enabled: !activityCenterMessage && !chatLogView.flickingVertically &&
```

Comment:
I think this is a good opportunity to remove the "global" accessing of properties (forgot the name). So maybe we can pass `chatLogView` from the parent as a prop instead.

Reply:
Yeah... even though I've seen it all over the place, I think it's fair to fix it (at least for this property). Will do

Reply:
@jrainville pls check and test as well, thanks :)